bokeh/_testing/util/env.py

```diff
@@ -0,0 +1,70 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2021, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+''' Provide functions for manipulating environment variables in tests.
+
+'''
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+import os
+from contextlib import contextmanager
+from typing import Any, Iterator, Mapping
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    'envset',
+)
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+@contextmanager
+def envset(value: Mapping[str, str]|None=None, **kw: Any) -> Iterator[None]:
```

Comment:
There should be spaces around `|`.

Reply:
Not until there is a lint rule for it and ideally an auto formatter that can be used. Working in other codebases with editor auto formatters makes it clear how much manual effort we waste ever day.